Alpaca.Markets/RestClient.cs

```diff
@@ -26,6 +28,12 @@ public sealed partial class RestClient : IDisposable
 
         private readonly String _polygonApiKey;
 
+        private readonly IEnumerable<Int32> _retryHttpStatuses;
+
+        private readonly String _alpacaApiVersion;
```

Comment:
Question - why do you use String not Int32 for version number?

Reply:
Its only purpose within the client is to be concatenated into another string, so it didn't provide any benefit to receive it Int32, and convert to String.

Reply:
Because string really contains any characters or can be even empty. Version number is a number and it's better to provide it as integer.